snippets/mcs-endpoint-limitation.adoc

```diff
@@ -0,0 +1,11 @@
+// Text snippet included in the following modules:
+//
+// * modules/about-crio.adoc
+// * modules/nodes-containers-using.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+Due to a known issue, it is recommended to use OpenShift SDN to secure the machine config server (MCS) endpoints: ports 22623 and 22624. Also, because of this issue, adding secrets to the ignition configuration file is not supported.
```

Comment:
also the first part - "use openshift SDN to secure the machine config server"
how?
what about OVN?

Reply:
I believe that OpenShiftSDN and OVN both block these ports from the pod network. I don't think any user intervention is needed. Can someone confirm this with the SDN team?

However, this does not block the port from the host network, I believe.

Reply:
@mburke5678 apologies for the delay here. How about this:
There is not a supported way to block or restrict the MCS endpoint in the current architecture. The MCS needs to be exposed to the network so that newly-provisioned machines, which have no existing configuration or state, are able to fetch their configuration. In this model, the root of trust is not at the MCS endpoint, but rather the CSR endpoint (this is where the kubelet sends its certificate signing request to be approved so that it can join the cluster), which means that Machine Configs cannot be used to distribute sensitive information. 

To make sure that MCS endpoints (accessible from ports 22623 and 22624) are secured in bare metal scenarios, customers are required to set-up proper network policies.

Reply:
@marrusl how does this work for you and Derrick?